libqfieldsync/offline_converter.py

```diff
@@ -317,9 +320,7 @@ def _convert(self, project: QgsProject) -> None:
                 ).transformBoundingBox(self.area_of_interest.boundingBox())
 
             is_success = self.offliner.convert_to_offline(
-                gpkg_filename,
-                offline_layers,
-                bbox,
+                gpkg_filename, offline_layers, bbox, self._export_title
```

Comment:
```suggestion
                gpkg_filename,
                offline_layers,
                bbox,
                self._export_title,
```

We prefer trailing comma on long lists to reduce the diff size.